### PR TITLE
Per-page reload opt-out

### LINKED ISSUE #241: Page-based opt-out of live refreshing
Feature request to add page-by-page checking for live refresh as [five-server](https://github.com/yandeu/five-server#watch--ignore) does.

### PR DESCRIPTION
**Pull request by ClintH**
TLDR; If the attribute `data-server-no-reload` is found on the `<body>` tag, commands to reload the page are ignored.

At the moment, all browser windows are reloaded when there is a source change, regardless of whether the source change affects them. This is a robust, base logic.

However, there are instances where one wants to prevent a page from being reloaded by changes. Eg, if _a.html_ opens up a connection to a media device, a change in _b.html_ will cause _a.html_ to be perhaps unnecessarily reloaded. This requires a manual re-starting of the stream, and other state may be lost.

This addition changes the injection script to check for the presence of an attribute on the `BODY` element. If it's there, the reload requests are ignored, and the page will have to be manually reloaded. The user can explicitly opt out of automatic reloading for pages they want manual control over, which otherwise is not currently possible.

It uses the same attribute name as [Five Server](https://github.com/yandeu/five-server) so the behaviour is identical regardless of which extension is used.

(Some other minor formatting changes happened applying the repo's eslint rules)

Fixes #241

### CODE REVIEW

**Review by andreamah — APPROVED** [review]
Awesome feature, thanks for adding this! I just added a little bit of code to ensure that the reload button in the embedded editor can work when the attribute prevents auto-refresh. :)

**Comment by ClintH** [comment]
Fantastic, thank you!

**Review by joyceerhl — APPROVED** [review]
